ietf/templates/doc/document_history.html

```diff
@@ -20,6 +20,11 @@
         <h2 class="my-3">Revision differences</h2>
         {% include "doc/document_history_form.html" with doc=doc diff_revisions=diff_revisions action=rfcdiff_base_url snapshot=snapshot only %}
     {% endif %}
+    {% if ballot_doc_rev %}
+        <p class="alert alert-info my-3">
+            Your most recent ballot position was provided on revision {{ ballot_doc_rev }}.
```

Comment:
nit: it would be more precise to say "was provided when the document was at version"